hapi/src/services/eden-historic-delegates.service.js

```diff
@@ -0,0 +1,158 @@
+const {
+  edenDelegatesGql,
+  edenHistoricElectionGql,
+  edenElectionGql
+} = require('../gql')
+const { hasuraUtil, sleepUtil, eosUtil, axiosUtil } = require('../utils')
+const { hyperionConfig, edenConfig } = require('../config')
+
+const historicDelegates = async ({
+  next_key: nextKey = null,
+  limit = 100
+} = {}) => {
+  return await eosUtil.getTableRows({
+    code: 'genesis.eden',
```

Comment:
load this value from `.env`

Reply:
Lets create a new env var `HAPI_EDEN_CONTRACT` so we may change the contract name if we deploy the code to a development environment such as jungle4 which has a different contract name for eden.